### ISSUE TITLE
Overwrite others sections in case there is an empty section title

### ISSUE THREAD

**Issue by byzheng** [kookma/TW-Section#30]
it might be a really wired situation.

Step to reproduce this bug

1. Go to website: https://kookma.github.io/TW-Section/ and create a new tiddler
2. modify tiddler title to `Section editor test` add a new field `se-type`, and save this tiddler.
3. Edit this tiddler and add content

```
!! Section 1

S1

!! 
```

There is a space in the empty section title.

![image](https://user-images.githubusercontent.com/1657381/174943442-73416e6b-eae8-496c-b3ed-978d561d9f84.png)

4. Save this tiddler 

![image](https://user-images.githubusercontent.com/1657381/174943551-50ed77c7-41c8-4556-b31c-2a93fe265f21.png)

5. Edit the second empty section add content

```
!! Section 2
S2
```

![image](https://user-images.githubusercontent.com/1657381/174943665-315c4a17-1733-4f67-be68-87b7ff83c2cf.png)

6. Save the section and the content changes into 

```
!! Section 2

S2Section 1

S1

!! Section 2

S2
```

![image](https://user-images.githubusercontent.com/1657381/174943764-0a55527b-55f4-429f-904a-023eeaac9b6e.png)

The content in `Section 1` is messed up.

I can confirm it is only happening when editing a section with empty title (i.e. `!! `)

**Comment by kookma** [comment]
Yes, this is a limitation in the current implementation!
Have you tried the above example with simple change:
When enter the body of tiddler for the second heading use two spaces e.g `!!<space><space>`
so your body looks like this

```
!! Section 1

S1

!!  
```

Note to the two spaces in front second `!!`

**Comment by byzheng** [comment]
I don't think it is a big problem for me. It is very easy to avoid  through writing the full section title before save it. 

I just thought it will be useful to report here.

**Comment by kookma** [comment]
There was some discussion on the forum on this. The recommendation is to use distinct header.
You may note if you create two exactly similar heading they overwrite each other. This is a limitation and I hope I can address this in future revisions
